src/test/java/data/TimeSheetCommonTest.java

```diff
@@ -13,16 +14,17 @@
     public void testInvalidTransferVacation() {
         ////Test values
         TimeSpan maxWorkingTime = new TimeSpan(40, 0);
-        TimeSpan vacation = new TimeSpan(10, 0);
         TimeSpan predTransfer = new TimeSpan(60, 0);
         TimeSpan succTransfer = new TimeSpan(20, 0);
         
         ////TimeSheet initialization
         Employee employee = new Employee("Max Mustermann", 1234567);
         Profession profession = new Profession("IPD", WorkingArea.UB, maxWorkingTime, 10.31);
         YearMonth yearMonth = YearMonth.of(2019, 11);
-        Entry[] entries = new Entry[0];
-        TimeSheet timeSheet = new TimeSheet(employee, profession, yearMonth, entries, vacation, succTransfer, predTransfer);
+        Entry[] entries = new Entry[] {
+            new Entry("Vacation", LocalDate.of(2019, 11, 29), new TimeSpan(0, 0), new TimeSpan(10, 0), new TimeSpan(0, 0), true)
```

Comment:
That test case isn't wrong but wouldn't it be clearer if we change the vacation's begin time to 10:00 and end to 20:00. At the moment it isn't 100% clear for a clueless reader why the test should fail (Time exceedance or vacation time out of legal bounds).

Reply:
There are no checks executed here, but I changed it for clarification.